opm/output/eclipse/VectorItems/doubhead.hpp

```diff
@@ -90,6 +91,12 @@ namespace Opm { namespace RestartIO { namespace Helpers { namespace VectorItems
 
         // Default => Use TSMINZ from TUNING
         constexpr auto NetBalMinTSDefault = 0.0;
+        
+        // Default => Let Simulator choose TSINIT
+        constexpr auto TSINITNoValue = 1.00000011;
```

Comment:
How did we select this threshold value (i.e., `TSINITNoValue`)?  It looks a little magical to me.

Reply:
Fairly arbitrary - just a value sufficiently close to 1 to pass tests. Feel free to suggest another value.